assets/js/amp-story-editor-blocks.js

```diff
@@ -39,6 +40,84 @@ var ampStoryEditorBlocks = ( function() { // eslint-disable-line no-unused-vars
 					value: 'lower-third',
 					label: __( 'Lower Third', 'amp' )
 				}
+			],
+			ampAnimationTypeOptions: [
```

Comment:
This list is duplicated with the array in `getAmpStoryAnimationControls`, is it not? I think there should be a common `getAnimationTypeOptions` function in the `utils` module to eliminate this duplication. Even if the list is not exactly the same, the function could still return the full list and then you could `filter` out the ones that aren't allowed.

Reply:
Yes, it is duplicated, however, the `utils.js` gets compiled, not sure what would be the best way to use the module like this within `amp-story-editor-blocks.js`, thoughts?

Created a separate helpers file for AMP Stories specifically within 4923569 since the `utils.js` always gets enqueued, however, `getAmpStoryAnimationControls` is needed for AMP Stories post type only, thought that then maybe we could add an inline script for defining the `getAnimationTypeOptions` globally to avoid duplication but not sure if that would be a good way to go.

Reply:
Oh, right. So this is actually dependent on #1298 which will convert `amp-story-editor-blocks.js` into a module. So once that happens then this problem would go away, right?